tools/esdlcmd/esdlcmd_common.hpp

```diff
@@ -142,6 +142,13 @@ typedef IEsdlCommand *(*EsdlCommandFactory)(const char *cmdname);
 #define ESDLOPT_INCLUDE_PATH_ENV        "ESDL_INCLUDE_PATH"
 #define ESDLOPT_INCLUDE_PATH_INI        "esdlIncludePath"
 #define ESDLOPT_INCLUDE_PATH_USAGE      "   -I, --include-path <include path>    Locations to look for included esdl files\n"
+#define ESDLOPT_RECURSIVE               "--recursive"
+#define ESDLOPT_RECURSIVE_S             "--r"
```

Comment:
Short names have one dash, long names have two.  So this should be simply "-r".

Reply:
will fix